x-pack/plugins/global_search/server/routes/find.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { reduce, map } from 'rxjs/operators';
+import { schema } from '@kbn/config-schema';
+import { IRouter } from 'src/core/server';
+
+export const registerInternalFindRoute = (router: IRouter) => {
```

Comment:
Though this endpoint is small, I think it'd still be good to have some unit tests on the error handling behavior.

Reply:
Added in https://github.com/elastic/kibana/pull/66293/commits/b66ccab167d005c4a4dd6e05a2e75ea4f1193a7b